pages/members/[slug].js

```diff
@@ -177,12 +203,57 @@ const MemberPage = ({ member, loadError }) => {
                 </Linkify>
               </p>
             }
+            </div>
+
+      <div className="flex flex-col items-start md:w-2/3">
+        <div>
+        <div className="page-title flex justify-between">
+          <h3 className="mt-16 mb-4">Upcoming events {member.screenname} is going to:</h3>
+          {/* <div className="action">
+            { member && member.roles.includes('event-creator') &&
+              <Link href="/events/create">
+                <a className="btn-primary">Create event</a>
+              </Link>
+            }
+          </div> */}
+        </div>
+        
+          <UpcomingEvents
+          allowCreate
```

Comment:
Fix indent please. But good catch we need to add the eslint rule - let's file a bug for later.